src/Raven.Server/ServerWide/Commands/DeleteDatabaseCommand.cs

```diff
@@ -81,6 +88,19 @@ public override void UpdateDatabaseRecord(DatabaseRecord record, long etag)
             record.Topology.Stamp.Index = etag;
         }
 
+        private void VerifyDeletion(DatabaseRecord record)
+        {
+            var databaseName = record.DatabaseName;
+            var topology = record.Topology;
+            foreach (var node in FromNodes)
+            {
+                if (topology.RelevantFor(node) == false)
```

Comment:
delete `Rehab` twice will not fail here, right? is that a problem?

Reply:
No, It will fail only when the node isn't in the topology anymore.. It's expected, so it isn't a problem

Reply:
So what will happened if I will request to delete `Rehab` node twice? 
How would the raft log will look like? one command with exception? or two commands without?

Reply:
Yes, one command will have an exception